torchtext/experimental/transforms/transforms.py

```diff
@@ -0,0 +1,86 @@
+from typing import List
+import torch
+from torchtext.data.functional import load_sp_model
+
+
+class SplitTokenizer(torch.nn.Module):
+    r""" Initiate a tokenizer transform class
+    """
+
+    def __init__(self):
+        super(SplitTokenizer, self).__init__()
+
+        def split_tokenizer(line: str) -> List[str]:  # noqa: F821
+            re_tokens: List[str] = []
+            for token in line.split():
+                if token != '':
+                    re_tokens.append(token)
+            return re_tokens
+        self.tokenizer = split_tokenizer
+
+    def forward(self, txt: str) -> List[str]:
+        return self.tokenizer(txt)
+
+
+class BasicEnglishTokenizer(torch.nn.Module):
+    r""" Initiate a tokenizer transform class
+    """
+
+    def __init__(self):
+        super(BasicEnglishTokenizer, self).__init__()
+
+        def basic_english_normalize(line: str) -> List[str]:
```

Comment:
Effectively this is the functional part of this transform and could be moved into such a file

Reply:
I agree.